README.md

```diff
@@ -89,7 +89,7 @@ let (p, mut c) = RabbitMqBackend::builder(cfg)
 	})
 	.with_decoder(|v: &Vec<u8>| -> Result<ExampleType, QueueError> {
 		Ok(ExampleType {
-			field: *v.first().unwrap_or(i&0),
+			field: *v.first().unwrap_or(&0),
```

Comment:
I forget what it's called, but I saw a crate that can treat code fences in your README.md as a doctest. Could help catch stuff like this.

Reply:
I'll look into adding this later. As of now, even the actual docs examples are marked `compile_fail` because it doesn't work with async out of the box.

Reply:
Made an issue for this here: https://github.com/svix/omniqueue-rs/issues/3

I'll mark this as resolved for now, but I intend to explore this possibility very soon.